molecule/logger.py

```diff
@@ -19,11 +19,27 @@
 #  DEALINGS IN THE SOFTWARE.
 
 import logging
+import os
 import sys
 
 import colorama
 
-colorama.init(autoreset=True)
+
+def should_do_markup(file=sys.stdout):
+    if os.environ.get('PY_COLORS') == '1':
+        return True
+    if os.environ.get('PY_COLORS') == '0':
+        return False
+    return hasattr(file, 'isatty') and file.isatty() and \
+        os.environ.get('TERM') != 'dumb' \
+        and not (sys.platform.startswith('java') and os._name == 'nt')
```

Comment:
Wow! Anyone runs in under jython?

Reply:
> os._name

I'd recommend avoiding accessing private attributes. Also, there's a `platform` stdlib module for this kind of env detection.


It's better to go for
```suggestion
        and platform.system() not in ('Java', 'Windows')
```

(of course `import platform` somewhere in the module header)